src/index.html

```diff
@@ -7,8 +7,58 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="index.html" class="logo">
+        <img
+        src="./images/logo.png"
+        alt="Moyo logo"
+        class="logo__img">
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a class="nav__link is-active "
+             href="#apple">Apple</a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#samsung">Samsung</a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#smartphones">Smartphones</a>
+          </li>
+
+          <li class="nav__item">
+            <a data-qa="hover"
+            class="nav__link"
+            href="#laptops"
+           >Laptops & Computers</a>
```

Comment:
<img width="357" alt="attrCheklist" src="https://github.com/mate-academy/layout_moyo-header/assets/77551820/1e7af747-c6c2-457e-b821-5601e3ce5324">

Reply:
Thank you for advice, I adjusted attributes. But I see that I passed tests. Maybe there is problem with link, it seems as it remember my first result when I didn't pass tests and doesn't update anymore.
<img width="975" alt="Screenshot 2023-12-29 at 10 41 51 AM" src="https://github.com/mate-academy/layout_moyo-header/assets/151964804/9ac1395a-cec7-49e1-950c-bd8a253cee19">

Reply:
Thank you for advice!